### ISSUE TITLE
Support for Edge (Chromium)

### ISSUE THREAD

**Issue by croban** [zmandel/Plus-for-Trello#18]
Can we expect "Plus for Trello" extension for Edge (Chromium). 
I have installed edge extension from chrome store just I can not sync with google sheet (stealth mode). Or is there some workaround? 
br

**Comment by zmandel** [comment]
Indeed, because Edge doesnt have Chrome sign-in, currently the chrome apis to access Google sheets cant work on Edge. Its doable but it would require to manually code the oauth2 flows needed (to get refresh and access tokens as needed).

**Comment by nagachaitanyalokam** [comment]
Hello @zmandel , 

Thank you for your interest to publish your extension on Edge Add-ons Website. 

If you have any more queries about publishing your extension to Microsoft Edge or anything else, you can contact the relevant teams whose links are given [here](https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/contact-extensions-team). 

Let me know if this was helpful. I am happy to provide you with more information.
